Playbooks/playbook-Access_Investigation_Generic_NIST_README.md

```diff
@@ -0,0 +1,72 @@
+
+Used Sub-playbooks:
+- IP Enrichment - Generic v2
+- Account Enrichment - Generic v2.1
+- Block IP - Generic v2
+- NIST - Lessons Learned
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+## Sub-playbooks
+* Block IP - Generic v2
+* NIST - Lessons Learned
+* Account Enrichment - Generic v2.1
+* IP Enrichment - Generic v2
+
+## Integrations
+* Builtin
+* Active Directory Query v2
+
+## Scripts
+* ADGetUser
+* GenerateInvestigationSummaryReport
+
+## Commands
+* setIncident
+* ad-disable-account
+* ad-expire-password
+* closeInvestigation
+* send-mail
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Required** |
+| --- | --- | --- | 
+| SrcIP | The source IP address from which the incident originated. | Optional |
+| DstIP | The target IP address that was accessed. | Optional |
+| Username | The email address of the account that was used to access the destionation IP address. | Optional |
+| NotifyEmail | The email addresses to notify about the incident. | Optional |
+| RemediationSLA | The Remediation SLA for the "Containment", "Eradication", and "Recovery" stage (in minutes). | Optional |
+| IPBlacklistMiner | The name of the IP address blacklist miner in MineMeld. | Optional |
+
+## Playbook Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| Account.Email.Address | The email address object associated with the account. | string |
+| DBotScore | The indicator, score, type, and vendor. | unknown |
+| Account.ID | The unique account DN (Distinguished Name). | string |
+| Account.Username | The account username. | string |
+| Account.Email | The email address associated with the account. | unknown |
+| Account.Type | The type of the account entity. | string |
+| Account.Groups | The groups the Account is part of. | unknown |
+| Account | The account object. | unknown |
+| Account.DisplayName | The account display name. | string |
+| Account.Manager | The account's manager. | string |
+| DBotScore.Indicator | The indicator value. | string |
+| DBotScore.Type | The indicator's type. | string |
+| DBotScore.Vendor | The indicator's vendor. | string |
+| DBotScore.Score | The indicator's score. | number |
+| IP | The IP address objects. | unknown |
+| Endpoint | The endpoint's object. | unknown |
+| Endpoint.Hostname | The hostname to enrich. | string |
+| Endpoint.OS | The endpoint OS. | string |
+| Endpoint.IP | The list of endpoint IP addresses. | unknown |
+| Endpoint.MAC | The list of endpoint MAC addresses. | unknown |
+| Endpoint.Domain | The endpoint domain name. | string |
+
+![Access_Investigation_Generic_NIST](https://github.com/demisto/content/blob/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Access_Investigation_Generic_NIST.png)
```

Comment:
This link is wrong. It should point to the **raw** path:

https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Access_Investigation_Generic_NIST.png

Need to fix this in all playbook docs